Source/RMG-Core/RomHeader.cpp

```diff
@@ -49,6 +140,8 @@ bool CoreGetCurrentRomHeader(CoreRomHeader& header)
     header.CRC2        = ntohl(m64p_header.CRC2);
     header.CountryCode = m64p_header.Country_code;
     header.Name        = CoreConvertStringEncoding((char*)m64p_header.Name, CoreStringEncoding::Shift_JIS);
-    
+    header.GameID      = generateGameID(m64p_header);
```

Comment:
One more nitpick, please don't capitalize letters in local functions, just name it `get_gameid_from_header` and `get_region_from_countrycode` like I showed in my example code.

Reply:
Done.